vsphere/panels.libsonnet

```diff
@@ -0,0 +1,1118 @@
+                ],
+              },
+            },
+          },
+          {
+            id: 'organize',
+            options: {
+              excludeByName: {},
+              includeByName: {},
+              indexByName: {
+                'Value #A': 3,
+                'Value #B': 4,
+                'disk_state 1': 2,
+                'vcenter_cluster_name 1': 1,
+                vcenter_datastore_name: 0,
+              },
+              renameByName: {
+                'Value #A': 'Memory usage',
+                'Value #B': 'Memory utilization',
+                disk_state: 'State',
+                'disk_state 1': 'State',
+                'disk_state 2': '',
+                'vcenter_cluster_name 1': 'Cluster',
+                vcenter_datastore_name: 'Datastore',
+              },
+            },
+          },
+        ]),
+
+      topCPUUsageResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top CPU usage resource pools',
+          targets=[t.topCPUUsageResourcePools],
+          description='The resource pools with the highest CPU usage.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('rotmhz'),
+
+      topMemoryUsageResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory usage resource pools',
+          targets=[t.topMemoryUsageResourcePools],
+          description='The resource pools with the highest memory usage.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('mbytes'),
+
+      topCPUShareResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top CPU share resource pools',
+          targets=[t.topCPUShareResourcePools],
+          description='The resource pools with the highest amount of CPU shares allocated.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('shares'),
+
+      topMemoryShareResourcePools:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top memory share resource pools',
+          targets=[t.topMemoryShareResourcePools],
+          description='The resource pools with the highest amount of memory shares allocated.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('shares'),
+
+      topCPUUtilizationEsxiHosts:
+        commonlib.panels.generic.timeSeries.base.new(
+          'Top CPU utilization ESXi hosts',
+          targets=[t.topCPUUtilizationEsxiHosts],
+          description='The ESXi hosts with the highest CPU utilization percentage.'
+        )
+        + g.panel.timeSeries.options.legend.withDisplayMode('table')
+        + g.panel.timeSeries.standardOptions.withUnit('percent'),
+
+      topMemoryUsageEsxiHosts:
```

Comment:
This should be a utilization panel, not usage

Reply:
Good catch